content/commands/json.mget/index.md

```diff
@@ -34,6 +34,8 @@ Return the values at `path` from multiple `key` arguments
 
 {{% warning %}}
 When cluster mode is enabled, all specified keys must reside on the same [hash slot](https://redis.io/docs/latest/operate/oss_and_stack/reference/cluster-spec/#key-distribution-model).
+
+When the database has more than one shard, and the specified keys reside in different shards, Redis will not report a CROSSSLOT error, and the results may be partial.
```

Comment:
maybe give some saddle hint as to why and what's expected in the future?

Reply:
@oranagra nothing was the very short term. For the longer term (see "How can we address this issue for the long term?" in https://redislabs.atlassian.net/browse/MOD-7315)

Reply:
i understand.. just suggesting that we add some hint in the text about that.
so users will have a clue that it was not intended, instead of asking why is it.